install_config/install/advanced_install.adoc

```diff
@@ -931,8 +931,8 @@ link:#verifying-the-installation[Verifying the Installation].
 == Verifying the Installation
 
 After the installer completes, you can verify that the master is started and
-nodes are registered and reporting in *Ready* status by running the following as
-*root*:
+nodes are registered and reporting in *Ready* status by sshing into the master
+host and running the following as *root*:
```

Comment:
Amended (but decided to add emphasis *On the master host*).

It also crossed my mind that switching to full `root@master.example.com#` prompts could help, as they convey exactly the desired information - which user on which host.  But that might be more confusing than useful unless it's applied consistently across docs...